### PR TITLE
Supervise

### PR DESCRIPTION
**Pull request by goodboy**
Rebase and fixes to port to new clearing system of #149.

@guilledk I think this is probably the smoothest way forward to avoid waiting on you to rebase 😂 

We can both work off this branch now.

### CODE REVIEW

**Comment by goodboy** [comment]
@guilledk so I think we can probably just land this since it's small enough and seems to mostly still work 😂 

the one last thing I was hoping to get going was running `pikerd` and being able to persist data feeds / shm arrays so that charts spawn faster when looking up an existing symbol and/or you've got alerts / orders that are still waiting to exec.

what you think?

File: piker/_daemon.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+"""
```
Comment (goodboy) [inline piker/_daemon.py:1]: Ahh we need license header..

File: piker/_daemon.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+# brokerd enabled modules
+_data_mods = [
+    'piker.brokers.core',
+    'piker.brokers.data',
+    'piker.data',
+    'piker.data._buffer'
+]
+
+
+async def spawn_brokerd(
```
Comment (goodboy) [inline piker/_daemon.py:94]: Do you think it's maybe just worth generalizing this spawn logic now?

For example we can have a `spawn_daemon(name: str)` and then just pass the module list in each case?

Comment (guilledk) [reply piker/_daemon.py:94]: Yeah that might be worth doing, easier to abstract shared behavoirs

**Comment by goodboy** [comment]
Need to cherry-pick in a few commits from cached_feeds branch still.

Will probably delay the spawn api formalizing for another PR just to start keeping things small.

**Comment by goodboy** [comment]
@guilledk pretty sure this is ready to land as well 🏄🏼

File: piker/_daemon.py [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+_root_dname = 'pikerd'
+_root_modules = [
+    __name__,
+    'piker.clearing._ems',
+    'piker.clearing._client',
+]
+
+
+# @dataclass
```
Comment (goodboy) [inline piker/_daemon.py:43]: heh lingering comment.

File: piker/_daemon.py (hunk 2 above)
Comment (goodboy) [reply piker/_daemon.py:131]: Kinda started this with the `_Services` type but we probably want to factor out the `spawn_<service>d()` code somewhat eventually as needed.

**Pull request merged**